src/main/java/controller/LottoGameController.java

```diff
@@ -42,13 +45,27 @@ private LottoMoney getLottoMoney() {
 		}
 	}
 
+	private Lottos buyLottos(LottoMoney lottoMoney) {
+		int manualLottoCount = lottoGameView.getManualLottoCount();
+		LottoMoney remain = lottoMoney.calculateRemainAfterBuy(manualLottoCount);
+
+		List<Lotto> manualLottos = getManualLottos(manualLottoCount);
+		List<Lotto> autoLottos = lottoGenerator.generateLottos(remain.calculateLottoCount());
+
+		return new Lottos(autoLottos, manualLottos);
```

Comment:
`manualLotto`인지 `autoLotto`인지는 Lottos 객체의 관심사가 아닐 것 같아요. 🤔
둘 다 그냥 Lotto이고 Lottos 객체는 Lotto만을위한 컬렉션이라고 생각해요. 🙂

아래와 각각의 로또뭉치들이 `Lottos` 타입을 가지고 두 Lottos를 합쳐주도록 변경해보면 어떨까요? 🙂
```suggestion
		Lottos manualLottos = getManualLottos(manualLottoCount);
		Lottos autoLottos = lottoGenerator.generateLottos(remain.calculateLottoCount());

		return manualLottos.addAll(autoLottos)
```

Reply:
현재 `Lottos`에는 다음과 같은 두 필드가 있습니다.
```java
private final List<Lotto> manualLottos;
private final List<Lotto> autoLottos;
```
수동인지 자동인지 여부를 어떤 필드 리스트의 요소로 존재하는지에 따라 구분한다고 볼 수 있는데요. 이렇게 만든 이유는 로또 게임에서 다음과 같이 수동과 자동 로또 개수를 구분해서 출력해야하기 때문입니다.
```java
System.out.printf("수동으로 %d개, 자동으로 %d개를 구매했습니다.\n",
	lottos.getManualLottoCount(), lottos.getAutoLottoCount());
```
이게 생성할 때부터 이렇게 구분해서 넣어주거나, 수동 및 자동 로또를 추가할 때 개수 필드를 따로 추가하지 않는 한 구분이 힘들어서 이렇게 두게 되었습니다.

Reply:
근데 말씀주신대로 `Lottos` 컬렉션은 수동인지 자동인지 여부를 알지 않아야한다는게 맞는 것 같습니다.
`Lotto`를 상속/구현하는 `AutoLotto`, `ManualLotto`같은걸 쓸 수도 있다는 생각이 드는데요,
일단 지금은 컨트롤러 단에서 `manualLottoCount`, `autoLottoCount`를 유지하여 뷰에 전달하는 식으로 리팩토링 했습니다!